src/main.py

```diff
@@ -77,8 +139,10 @@ async def help(update, context):
     /start - Start the bot + register the club
     /run_club_polling - Run polling for the club
     /stop_club_polling - Stop polling for the club
-    /add_team - We should recognise message_thread_id of the topic as it unique ID for general 0. Same chat_id for all topics. I haven't investigated it deeply. In any case, we need to attach team to the club (with oportunity to send msgs to specific topic), initialize it with a title, team_id, topic_id
+    /add_team <team_id> - Add team with specific team_id to the club
+    /remove_team <team_id> - Remove team with specific team_id to the club
     """
+    # TODO: should the following line be replaced with ChatHelper instance method?
```

Comment:
Ideally we always have to use `ChatHelper` instead of `send_message`  and `update.message.reply_text`